src/index.html

```diff
@@ -11,7 +11,131 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="box">
+        <legend>
+          Personal information
+        </legend>
+        <label class="form-field">
```

Comment:
```suggestion
        </legend>
        
        <label class="form-field">
```
It is better to add new line between adjacent elements. Please fix all cases